lib/content/gittenify.js

```diff
@@ -0,0 +1,33 @@
+import catNames from 'cat-names';
+import mutationSummary from 'mutation-summary';
+import catPhoto from './catPhoto';
+import replaceUsers from './replaceUsers';
+
+export default function gittenify(store) {
+  const userMap = {};
+
+  for (const u of store.users) {
```

Comment:
That's what I meant by "Semantically". While the variable inside the block itself doesn't change, the variable "itself" does. If it checks out, it's fine. Just a bit odd ;)